crates/shared/src/sources/balancer_v2/event_handler.rs

```diff
@@ -322,3 +322,90 @@ fn convert_stable_pool_created(
         },
     })
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::{
+        sources::balancer_v2::{
+            info_fetching::PoolInfoFetcher,
+            pool_init::{EmptyPoolInitializer, SubgraphPoolInitializer},
+        },
+        token_info::TokenInfoFetcher,
+        transport,
+    };
+    use contracts::BalancerV2Vault;
+    use maplit::hashset;
+    use reqwest::Client;
+
+    #[tokio::test]
+    #[ignore]
+    async fn balancer_index_pool_events() {
```

Comment:
nanonit: name could be a bit clearer that this is comparing indexed vs subgraph events.

Reply:
Fixed!